test/parallel/test-fs-null-bytes.js

```diff
@@ -1,21 +1,20 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
 
 function check(async, sync) {
-  var expected = /Path must be a string without null bytes/;
-  var argsSync = Array.prototype.slice.call(arguments, 2);
-  var argsAsync = argsSync.concat(function(er) {
+  const expected = /Path must be a string without null bytes/;
```

Comment:
@lpinca this test is using that variable twice to validate the error, one of this is an `assert.throws` if I set the  boundaries that one will fail, I could use 2 different vars to test the error... but... not sure if worth it?